docs/PLATFORMS.md

```diff
@@ -0,0 +1,68 @@
+openFrameworks currently supports the following platforms:
+
+# Android #
+
+versions: 2.2+
+
+architectures:
+  * armeabi-v7a
+  * x86 (i686)
```

Comment:
i think it's better to specify x86 only, in all the platforms, i686 is correct in the case of android but i386 specified in other places is surely not compatible with the libraries we ship